rootmap/rootmap_test.go

```diff
@@ -86,22 +207,106 @@ func TestRootmaps(t *testing.T) {
 					converted = append(converted, listing)
 				}
 				require.ElementsMatch(t, []rootmap.RootListing{
-					{testPrefix, "my-device", "topic1", node1, 40, "", 0},
-					{testPrefix, "my-device", "topic2", node2, 50, "", 0},
+					{"test", "my-device", "topic1", node1, 40, "", 0},
+					{"test", "my-device", "topic2", node2, 50, "", 0},
 				}, converted)
-			})
-			t.Run("get version that does not exist", func(t *testing.T) {
-				_, _, err := rm.Get(ctx, "db", "fake-device", "my-topic", 1e9)
-				require.ErrorIs(t, err, rootmap.TableNotFoundError{})
-			})
-			t.Run("get latest version that does not exist", func(t *testing.T) {
-				_, _, _, err := rm.GetLatest(ctx, "db", "fake-device", "my-topic")
-				require.ErrorIs(t, err, rootmap.TableNotFoundError{})
-			})
+			},
+		},
+		{
```

Comment:
can we have an explicit case for when the requested version does and does not exceed the truncation version? Also I wonder if any of this test content can perhaps be split into helpers, seems like the bodies of the case statements will be long

Reply:
Sure!